src/v/kafka/server/handlers/fetch.cc

```diff
@@ -475,7 +475,7 @@ static void fill_fetch_responses(
         resp_it->set(std::move(resp));
         std::chrono::microseconds fetch_latency
           = std::chrono::duration_cast<std::chrono::microseconds>(
-            start_time - op_context::latency_clock::now());
+            op_context::latency_clock::now() - start_time);
```

Comment:
Ooops, of course "now" is later than "start_time" at least in this universe.

I tested this locally with the ops dashboard in Grafana (which also reproduced the old problem) and latencies dropped to < 1 ms. I did test my old change against my local Grafana but it seems like there is no panel for fetch latency on the dashboard I was using, so this slipped by.